api/soap/mantisconnect.wsdl

```diff
@@ -163,6 +164,8 @@
    <xsd:element name="monitors" type="tns:AccountDataArray" minOccurs="0"/>
    <xsd:element name="sticky" type="xsd:boolean" minOccurs="0"/>
    <xsd:element name="tags" type="tns:ObjectRefArray" minOccurs="0"/>
+   <xsd:element name="alias" type="xsd:string" minOccurs="0"/>
```

Comment:
Not really.  Again I want to reduce surface API of the feature in the public API and also I kind of think of the alias/hash as something that doesn't need to be displayed or returned.